edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter/handlers/ConnectionHandler.cs

```diff
@@ -200,6 +216,27 @@ async Task AddConnectionsAsync(HashSet<IIdentity> identitiesAdded)
             }
         }
 
+        async Task ReplaceConnectionsAsync(HashSet<IIdentity> identitiesReplaced)
+        {
+            var connectionProvider = await this.connectionProviderGetter;
+            if (connectionProvider == null)
+            {
+                Events.FailedToObtainConnectionProvider();
+                return;
+            }
+
+            foreach (var identity in identitiesReplaced)
+            {
+                if (this.knownConnections.TryRemove(identity, out var deviceListener))
+                {
+                    // AddConnectionAsync() would do this, but let's avoid the warning
```

Comment:
Why is the event a warning? It kinda seems like this could be a very normal change/use case?

Reply:
yes, so at that time it seemed suspicious and was implemented with the warning, but you are right that best would be to remove the warning and let the AddConnectionAsync() do its work